e2e/tests/functional/plugins/lad/ladSet.e2e.spec.js

```diff
@@ -53,6 +53,9 @@ test.describe('LAD Table Sets', () => {
 
     await page.goto(ladTableSet.url);
 
+    // Wait for the initial value to show after mount
+    await expect(page.getByLabel('lad value').first()).not.toContainText('---');
```

Comment:
We will need to factor out the use of first at some point

Reply:
I think it's ok if we're accessing a value in a table?